src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -324,8 +321,7 @@ class FileField extends React.Component {
               );
               const attachmentNameErrors = get([index, 'name'], errorSchema);
               // feature flag
-              const showPasswordContent =
-                requestLockedPdfPassword && file.isEncrypted;
+              const showPasswordContent = file.isEncrypted;
```

Comment:
The `// feature flag` comment above can go now. Additionally, with the feature flag gone `showPasswordSuccess` seems a little hard to follow now:

https://github.com/department-of-veterans-affairs/vets-website/blob/092178c895ca9bfa0acbf084c45b698f2e04373d/src/platform/forms-system/src/js/fields/FileField.jsx#L329-L334

If `showPasswordContent` is just `file.isEncrypted`, then we basically have:

```js
const showPasswordInput =
    file.isEncrypted && !file.confirmationCode;
const showPasswordSuccess =
    file.isEncrypted &&
    !(file.isEncrypted && !file.confirmationCode) &&
    file.confirmationCode;
```

That middle boolean expression might not be necessary anymore since both `isEncrypted` and `confirmationCode` have to be truthy for the whole expression to be true. I think we could simplify to:

```js
const showPasswordInput =
    file.isEncrypted && !file.confirmationCode;
const showPasswordSuccess =
    file.isEncrypted && file.confirmationCode;
```

Reply:
Good catch! I cleaned up the file a little bit, which I _think_ should address your comment. If not, let me know.

Reply:
Looks good to me :+1: